packages/fast-components-react-base/src/context-menu-item/context-menu-item.props.ts

```diff
@@ -25,6 +25,15 @@ export interface ContextMenuItemHandledProps extends ContextMenuItemManagedClass
      * If the menu item is disabled
      */
     disabled?: boolean;
+
+    /**
+     * Callback for when an item is invoked
+     * Returns the prop contract for the invoked context menu item
+     */

```

Comment:
I think you want props to not be optional, otherwise you're allowing us to internally call the function without the props argument.